examples/files/bobby_phones_elan.TextGrid

```diff
@@ -0,0 +1,74 @@
+File type = "ooTextFile"
+Object class = "TextGrid"
```

Comment:
I've never used ELAN, but does it write point tiers too?  If so, could you add one to this Textgrid.

Reply:
I also have never used ELAN so I don't know for sure if it exports point tiers or what format they're in.  I'll do a bit of research, but I can add a similarly formatted point tier test case just to cover the relevant aspects.

Reply:
Thanks for adding the PointTier in 5e3a2f8.  I should be able to do one more look through in a bit.  If all is good, I'll merge and make a release.  Thanks!